.github/workflows/monorepo_actor_release.yml

```diff
@@ -0,0 +1,146 @@
+    #   secret_output: 'place-holder'
+    steps:
+      - uses: actions/checkout@v2
+      - uses: wasmcloud/common-actions/install-wash@main
+      - name: Add wasm32-unknown-unknown
+        run: rustup target add wasm32-unknown-unknown
+      # Once you've written unit tests for your actor, you can uncomment
+      # the two lines below to automatically run tests
+      # - name: Test actor
+      #   run: cargo test --target x86_64-unknown-linux-gnu -- --nocapture
+      - name: Extract Actor Name from Tag
+        run: |
+          tag=${GITHUB_REF#refs/tags/*-v*/}
+          tag2=${tag##*\/}
+          echo "ACTOR_NAME=${tag2%-v*}" >> $GITHUB_ENV
+      - name: Extract Actor Version from Tag
+        run: |
+          tag=${GITHUB_REF#refs/tags/*-v*/}
+          version=`expr match "$tag" '.*\([0-9][0-9]*\.[0-9][0-9]*\.[0-9][0-9]*\)'` 
+          echo "ACTOR_VERSION=${version}" >> $GITHUB_ENV
+      - name: Set working Directory
+        run: |
+          echo "WORKING_DIR=$PWD/actor/$ACTOR_NAME" >> $GITHUB_ENV
+      - name: Determine Actor Name from Cargo
+        working-directory: ${{ env.WORKING_DIR }}
+        run: |
+          echo "VERIFY_ACTOR_NAME=$(cargo metadata --no-deps --format-version 1 | jq -r '.packages[].name')" >> $GITHUB_ENV
+      - name: Determine Actor Version from Cargo
+        working-directory: ${{ env.WORKING_DIR }}
+        run: |
+          echo "VERIFY_ACTOR_VERSION=$(cargo metadata --no-deps --format-version 1 | jq -r '.packages[].version')" >> $GITHUB_ENV
+      - name: Verify Cargo and Tag are in Sync
+        run: |
+          if ${ACTOR_NAME} == ${VERIFY_ACTOR_NAME} && ${ACTOR_VERSION} == ${VERIFY_ACTOR_VERSION}; then
+            echo "VERIFIED=true" >> $GITHUB_ENV
+          else
+            echo "VERIFIED=false" >> $GITHUB_ENV
+          fi
+      - name: Set Verification Output
+        id: set_verified
+        run: |
+          if ${VERIFIED} == 'true'; then
+            echo "::set_output name=verification::true" >> $GITHUB_OUTPUT
+          else
+            echo "::set_output name=verification::false" >> $GITHUB_OUTPUT
+            echo "::error title=VerificationFailed::Cargo Version Does Not Match Tag Version"
+          fi
+      - name: Build Subject Key Secret String
+        if: env.VERIFIED == 'true'
+        run: |
+         secret_prefix='WASH_'
+         secret_unique=$(echo $ACTOR_NAME | tr 'a-z' 'A-Z')
+         secret_suffix='_KEY'
+         secret=${secret_prefix}${secret_unique}${secret_suffix}
+         echo "SECRET_STRING=$secret" >> $GITHUB_ENV
+      - name: Build wasmcloud actor
+        if: env.VERIFIED == 'true'
+        env:
+          WASH_SUBJECT_KEY: ${{ secrets[env.SECRET_STRING] }}
+        run: wash build
+        working-directory: ${{ env.WORKING_DIR }}
+      - name: Upload signed actor to GH Actions
+        if: env.VERIFIED == 'true'
+        uses: actions/upload-artifact@v2
+        with:
+          name: $ACTOR_NAME
```

Comment:
Note: I would like to go through and unify env var reference as much as possible prior to completion. There may be some instances where references to variables can be shifted to a more consistent option. Currently, I use `env.`, `$` and context `${{}}`. Can't likely change everything but I would like to go towards more consistency.